derp/emulator.py

```diff
@@ -136,3 +180,5 @@ def setup_filter_on_healpix(region):
 
 def filter_on_healpix(hp, list_of_healpix):
     return np.array([hh in list_of_healpix for hh in hp])
```

Comment:
Depending on the sizes of `hp` and `list_of_healpix`, it might be faster to do `set(list_of_healpix) & set(hp)` since they are unordered. Is the conversion to `np.array` necessary? Or was there a complication when using it in the lambda function for filtering? But this is super minor and comes close to mattering only when we run `filter_on_healpix` many times.

Reply:
:-) Let's try to remember this when we get to scaling up - I guess we'd see the healpix filtering in the profile at that point too.